react-events/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,14 @@
+import './CustomButton.css';
+
+export default function CustomButton({ text, color, onCustomClick }) {
+  function handleClick() {
+    onCustomClick(text);
+  }
+  return (
+    <>
+      <button className={color} onClick={handleClick}>
```

Comment:
you can also use a anonymous call back function inside of the onClick code block